hazelcast-jet-hadoop/src/main/java/com/hazelcast/jet/impl/connector/hadoop/WriteHdfsP.java

```diff
@@ -161,18 +169,15 @@ public void init(@Nonnull Context context) {
 
         @Override
         public void complete(Throwable error) {
-            if (commitJob) {
-                uncheckRun(() -> outputCommitter.commitJob(jobContext));
+            if (processorList != null) {
+                processorList.forEach(Processor::complete);
```

Comment:
wouldn't this cause Processor.complete() to be called twice? Once after DAG finishes, and once on PS.complete()?

Reply:
Yes, I've missed to change it to a private `close` method, fixed now.